tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/InstrumentationTests.cs

```diff
@@ -251,6 +251,21 @@ public async Task WhenUsingPathWithDotsInInTracerHome_InstrumentsApp()
             agent.Telemetry.Should().NotBeEmpty();
         }
 
+        [SkippableFact]
+        [Trait("RunOnWindows", "True")]
+        public async Task WhenOmittingTracerHome_InstrumentsApp()
+        {
+            SetLogDirectory();
+
+            // DD_DOTNET_TRACER_HOME is not set, so the tracer should derive it from the profiler path
+            Output.WriteLine("DD_DOTNET_TRACER_HOME not set, relying on profiler path environment variables");
```

Comment:
Should we / Could we check to assert that it isn't set and fail the test if it is set? Wondering about chance of it getting picked up from other tests, but not really sure if that can happen in these.